src/CharterManager.sol

```diff
@@ -229,6 +242,28 @@ contract CharterManagerImp {
         VatLike(vat).move(urp, dst, rad);
     }
 
+    function roll(bytes32 srcIlk, bytes32 dstIlk, address src, address dst, uint256 srcDart) external allowed(src) allowed(dst) {
```

Comment:
I don't think we need to allow the `src` as basically it is getting its debt repaid. It could be permissionless.

Reply:
Yeah I know, was trying to be strict here.
The use case is for an institutionval vault owner which control both vaults.
Can change it if we think it's worth it.

Reply:
Yeah fair enough, let's leave it as it is.